src/dotenv-azure.ts

```diff
@@ -194,10 +202,10 @@ export default class DotenvAzure {
     )
   }
 
-  private getAzureCredentials(dotenvVars: DotenvParseOutput = {}): AzureCredentials {
+  private getAzureCredentials(dotenvVars: Record<string, string | undefined> = {}): AzureCredentials {
     const vars = { ...dotenvVars, ...process.env }
```

Comment:
I think we could delete this line since you are already concatenating the `process.env` variables on line 63